### ISSUE TITLE
Status of this project

### ISSUE THREAD

**Issue by ctrueden** [sdedwards/m2e-nar#1]
@sdedwards Are you actively using/maintaining this project? Is it working and stable? If so, would you be interested in moving it upstream (https://github.com/maven-nar), and adding it to the documentation, to increase its visibility for the NAR community?

**Comment by ctrueden** [comment]
To be clear, I'm suggesting this to make the project more visible, _not_ to hand off maintenance to anybody else.

**Comment by sdedwards** [comment]
I'm still using it yes. It's working and stable for gcc compatible compilers on MacOS and Linux. It'll need updating for Windows compilers, which I should have time to do within the next few weeks if I can get the environment set up.

Thanks for the offer, I would be happy to move it! How do we go about this?

**Comment by ctrueden** [comment]
> Thanks for the offer, I would be happy to move it! How do we go about this?

I have to run right this moment, but I will add you to the NAR organization later tonight, and then we'll see if it's possible for you to perform the transfer (from the Settings menu of the repository).

**Comment by dproteus13** [comment]
+1

**Comment by ctrueden** [comment]
@sdedwards I invited you to the maven-nar organization. I also changed the membership permissions such that all NAR members have write access to all repositories, as well as the ability to create new repositories. I am hoping this gives you sufficient privileges to transfer this `m2e-nar` repository into that org. Hopefully you'll retain admin access, but if not (i.e., if it warns you that you will lose admin access), then we'll just restore that access post-transfer.

Let me know how it goes!

**Comment by sdedwards** [comment]
> Let me know how it goes!

The transfer fails with:
`You don’t have the permission to create repositories on maven-nar `
Could you please give me the required permissions?

**Comment by ctrueden** [comment]
> Could you please give me the required permissions?

Sorry about that; somehow the "Allow members to create repositories for this organization" option was still off. I turned it on again and it appears to have persisted this time. Try again?

**Comment by sdedwards** [comment]
Done! I've lost the settings tab now though, could you restore admin access please?

**Comment by ctrueden** [comment]
@sdedwards Done! All good now?

I think the next step is to update the NAR documentation (maybe starting with the [wiki](https://github.com/maven-nar/nar-maven-plugin/wiki)?) with a page about `m2e-nar`?

**Comment by sdedwards** [comment]
Yes thanks, all good. I'll start working on some documentation as soon as I can.
